test/test_environment.lua

```diff
@@ -210,7 +209,7 @@ function test_env.set_args()
    if not test_env.TEST_TARGET_OS then
       title("OS CHECK")
 
-      if execute_bool("sw_vers") then
+      if execute_bool("sw_vers") then 
```

Comment:
whitespace at end of line

Reply:
I actually put that back in this commit to revert a whitespace-only diff in the full PR, so the spurious change doesn't get in when the PR is squashed.

(I mind the noise of whitespace-only diffs way more than I do the actual presence of trailing whitespace. :) )